package.json

```diff
@@ -97,7 +97,10 @@
     "typescript": "3.7.5"
   },
   "dependencies": {
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
+    "react-redux": "^7.0.0",
```

Comment:
By having react-redux as a specific version here it means having a newer major version (e.g. v8) in the application causes issues. See https://react-redux.js.org/troubleshooting#could-not-find-store-in-either-the-context-or-props